config/mappings/nstf_mapping_parts/nstf_r2_scored_criteria.py

```diff
@@ -551,18 +551,26 @@
                                 "question": "Which funding are you applying for?",
                             },
                             {
-                                "field_id": "pVBwci",
+                                "field_id": ["pVBwci", "GRWtfV"],
                                 "form_name": "funding-required-ns",
                                 "field_type": "numberField",
-                                "presentation_type": "text",
-                                "question": "How much revenue are you applying for? 1 April 2023 to 31 March 2024",
+                                "branched_field": "1",
+                                "presentation_type": "grouped_fields",
+                                "question": [
+                                    "How much revenue are you applying for? 1 April 2023 to 31 March 2024",
+                                    "How much revenue are you applying for? 1 April 2023 to 31 March 2024",
+                                ],
                             },
                             {
-                                "field_id": "WDouQc",
+                                "field_id": ["zvPzXN", "WDouQc"],
+                                "branched_field": "1",
+                                "presentation_type": "grouped_fields",
                                 "form_name": "funding-required-ns",
                                 "field_type": "numberField",
-                                "presentation_type": "text",
-                                "question": "How much revenue are you applying for? 1 April 2024 to 31 March 2025",
+                                "question": [
+                                    "How much revenue are you applying for? 1 April 2024 to 31 March 2025",
+                                    "How much revenue are you applying for? 1 April 2024 to 31 March 2025",
```

Comment:
Are these meant to be duplicates?

Reply:
yes its a requirement for a grouped field. ones the title ones the caption, see: https://github.com/communitiesuk/funding-service-design-assessment/blob/f46c335016858c65302c2c46be9a236d93d59c7f/app/assess/models/ui/applicants_response.py#L121